src/components/Layout/Footer.tsx

```diff
@@ -1,15 +1,70 @@
-import React, { FunctionComponent } from "react";
+import React from "react";
+import { Footer as BaseFooter, FooterColumnItemProps } from "@govtechsg/tradetrust-ui-components";
+import { NavLink } from "react-router-dom";
+import { ExternalLink } from "react-feather";
+import tw from "twin.macro";
+import styled from "@emotion/styled";
+import { URLS } from "../../constants";
 
-export const Footer: FunctionComponent = () => {
+const sharedStyles = `text-cloud-500 hover:text-black transition-color duration-200 ease-out font-medium text-sm`;
+const renderNavLink = ({ label, to }: FooterColumnItemProps) => {
   return (
-    <footer className="bg-white py-6 no-print">
-      <div className="container">
-        <div className="flex justify-center">
-          <div className="w-auto">
-            <p className="mb-0">Copyright &copy; 2020 TradeTrust</p>
-          </div>
-        </div>
+    <NavLink className={sharedStyles} to={to}>
+      {label}
+    </NavLink>
+  );
+};
+const renderExternalLink = ({ label, to }: FooterColumnItemProps) => {
+  return (
+    <a className="flex items-center" href={to} target={"_blank"} rel="noopener noreferrer">
+      <p className={`${sharedStyles} mr-1`}>{label}</p>
+      <div className={`w-auto`}>
+        <ExternalLink size={12} color={"#89969F"} />
       </div>
-    </footer>
+    </a>
   );
 };
+
+const data = [
+  {
+    category: "Utilities",
+    items: [
+      { label: "Verify Documents", to: "/verify", render: renderNavLink },
+      { label: "Create Documents", to: URLS.CREATOR, render: renderExternalLink },
+    ],
+  },
+  {
+    category: "Resources",
+    items: [
+      { label: "Webinars", to: "B-1", render: renderExternalLink },
```

Comment:
know the location in the future @simboonlong ?

Reply:
if im not wrong, webinars is `/learn`

Reply:
👌